### DISCUSSION TITLE
Update to lowest minor when updating major

### DISCUSSION THREAD

**Discussion by jpgoelz** [renovatebot/renovate#25449] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I am having the following issue:

Our project is using Angular 16 and was using TypeScript v4.8.4.

When using `:separateMajorReleases` and `:separatePatchReleases`, it suggests one update to 4.9.5, which is fine and an update to 5.2.2 which does not work because Angular requires TS < 5.2.

When applying the update to 4.9.5, the next update still is 5.2.2.

What can I do so that Renovate suggests an update to maybe 5.0.4 or 5.1.6 first? Basically, I would like an option to not update to the latest major or minor versions.

The same also applies to updates for Spring Boot, where Renovate now suggests updating from 2.7 to 3.1. Where it may be important to do the in between migration step to 3.0 first, because most migration guides do not cover skipping steps.

### What you tried so far.

I searched the docs as best as I could.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by jpgoelz** [comment]
Maybe this goes well together with options similar to `:separateMultipleMajorReleases` ([docs](https://docs.renovatebot.com/presets-default/#separatemultiplemajorreleases)), like `:separateMultipleMinorReleases:`, which does not yet exist.

**Comment by viceice** [comment]
not yet possible, but there is an open PR for separate multiple minor.

**Reply by rarkins** [reply]
I'm not sure it still separates for major updates
